### PR TITLE
INSTUI-3773 clean up `ui-scripts`

### PR DESCRIPTION
**Pull request by matyasf**
- remove not needed functions
- convert every command to `yargs`
- add delay to all batch `npm` commands to prevent errors
- removed deprecated commands that just resulted in a warning

to test: run the changed commands

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,10 +87,6 @@
     "yarn": ">=3"
   },
   "config": {
-    "ui-scripts": {
-      "npm_scope": "@instructure:registry=https://registry.npmjs.org/",
-      "changelog_url": "https://instructure.design/#CHANGELOG"
-    },
```
Comment (matyasf) [inline package.json:94]: this is the config of ui-scripts. changelog_url is unused now, I've moved the other to a constant where its used.

File: package.json [hunk 2]
```diff
@@ -46,8 +46,7 @@
     "clean": "yarn node scripts/clean.js",
     "export:icons": "lerna run export --stream --scope @instructure/ui-icons",
     "bump": "ui-scripts bump",
-    "link:packages": "ui-scripts --link-packages",
-    "release": "ui-scripts --publish",
+    "release": "ui-scripts publish",
```
Comment (matyasf) [inline package.json:49]: i had to remove `--` from params because `yargs` doesnt like them

File: packages/ui-scripts/package.json [hunk 3]
```diff
@@ -18,32 +18,29 @@
   },
```
Comment (matyasf) [inline packages/ui-scripts/package.json:18]: I actually really dislike that `ui-scripts` has 3 differently named binaries. What do you think of keeping just one (e.g. `ui-test --randomize` will be `ui-scripts test --randomize`) ? (in a later PR)

Comment (HerrTopi) [reply packages/ui-scripts/package.json:18]: I think this is a very good idea. Lets have a task about it!

Comment (matyasf) [reply packages/ui-scripts/package.json:18]: 👍 ill create one

**Review by HerrTopi — APPROVED** [review]

**Pull request merged**